build-caching-maven-samples/protobuf-project/pom.xml

```diff
@@ -32,6 +32,7 @@
             </goals>
             <configuration>
               <outputDirectory>target/generated-sources/protobuf</outputDirectory>
+              <protocArtifact>com.google.protobuf:protoc:3.17.3</protocArtifact>
```

Comment:
Maybe worth commenting that this is required for M1 support

Reply:
Added in e978c05e9a080036d02ff34d4d3f2d302a76ce20